src/engine/ruleset/decoders/wazuh-agent/syscollector/syscollector-dbsync.yml

```diff
@@ -0,0 +1,108 @@
+name: decoder/syscollector-dbsync/0
+
+metadata:
+    module: wazuh-agent/syscollector/syscollector-dbsync
+    title: Syscollector Decoder event for dbsync
+    description: >
+        Third stage filter of syscollector event decoder with type "dbsync_"
+        if succes will query to wdb and map on "wazuh.syscollector..."
+        -> only for wazuh managers with version 4.4 and above.
+    compatibility: >
+        This decoder has been tested on Wazuh version 4.4.
+    author:
+        name: Wazuh, Inc.
+        url: https://wazuh.com
+        date: 2023/03/08
+    references:
+        - https://documentation.wazuh.com/current/user-manual/capabilities/syscollector.html#using-syscollector-information-to-trigger-alerts
+        - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/wodle-syscollector.html
+        - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-db.html#syscollector-tables
+
+sources:
+  - decoder/syscollector-dbsync-base/0
+
```

Comment:
Is it possible to unify this decoder with the db-sync-base?

Reply:
It could be done but it's also used on the other syscollector decoders (not the dbsync ones)